### PR TITLE
BUG: Fix build archs

### PR DESCRIPTION
**Pull request by carterbox**
On Windows, in the bld.bat, the extra archs for CUDA >=11.0 aren't actually being appended to that environment variable. This PR aims to fix this issue. Check the CI logs for the NVCC_FLAGS line of the CMAKE output to see if it is working correctly.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
hmm, i don't know too much about these things.

This website says that 11.1 should support 86 too.
https://arnon.dk/matching-sm-architectures-arch-and-gencode-for-various-nvidia-cards/

File: recipe/build.sh [hunk 1]
```diff
@@ -6,11 +6,12 @@ export PATH=$PREFIX/bin:$PATH
 
 export CUDA_ARCH_LIST="-gencode arch=compute_37,code=sm_37 -gencode arch=compute_50,code=sm_50 -gencode arch=compute_60,code=sm_60 -gencode arch=compute_70,code=sm_70"
```
Comment (carterbox) [inline recipe/build.sh]: Use this website for determining which archs to build for?

https://arnon.dk/matching-sm-architectures-arch-and-gencode-for-various-nvidia-cards/

**Comment by carterbox** [comment]
> This website says that 11.1 should support 86 too.

This build script agrees with [the website](https://arnon.dk/matching-sm-architectures-arch-and-gencode-for-various-nvidia-cards/). It adds sm_86 for CUDA>=11.1.

**Comment by hmaarrfk** [comment]
> sm_86 for CUDA>=11.1.

I think i confused 11.0 with 11.1 when i first read your PR.

**Review by hmaarrfk — APPROVED** [review]

**Pull request merged**